### DISCUSSION TITLE
Is the information from @db annotations stored in the dmmf?

### DISCUSSION THREAD

**Discussion by chrishoermann** [prisma/prisma#17684] [Q&A]
I'd love to have the possibility of creating zod validators in my generator ([zod-prisma-types](https://github.com/chrishoermann/zod-prisma-types)) that read out the information of the @db annotations of the prisma schema.

for example when `field String @db.Uuid` is used I want to generate the zod schema 

```ts
const MyModelSchema = z.object({
  field: z.string().uuid()
})
```

Is the information stored somewhere in the dmmf?

**Comment by janpio** [comment]
Unfortunately not yet: https://github.com/prisma/prisma/issues/6844
But we are currently working on making that possible in the future. We absolutely understand that use case.

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that haven’t had any recent activity and appear to be outdated.

If this discussion is still important to you or unresolved, we’d love to hear from you! Feel free to reopen it or start a new one with updated details.

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and being part of the community!
